docs/design/fabric-v2+/fpc-integration-touchpoints.puml

```diff
@@ -63,19 +64,33 @@ group Per FPC chaincode FCC
    note over Client1
      extend CC definition with additional meta-data
    end note
+   Client2 -> Client2: chaincode package FPC
+   
+   Client1 -> Peer1: chaincode install FPC
+   Client2 -> Peer2: chaincode install FPC
 
    Client1 -> Peer1 : chaincode approveformyorg FPC
-   Client1 -> Peer2 : chaincode approveformyorg FPC
+   Client2 -> Peer2 : chaincode approveformyorg FPC
 
    Client1 -> Peer1 : chaincode commit FPC
    note over Peer1
      trigger preparation of execution enclaves & key-distribute for all peers
    end note
    Peer1 <-> Peer2
-   & Peer2 <-> ecc1 
-   & ecc1 <-> ecc2
+
+   Client1 -> Peer1 : chaincode initEnclave FPC
+   & Peer1 <-> ecc1 
    activate ecc1
-   activate ecc2
+
+   alt "full" FPC (Post-MVP)
+      Client2 -> Peer2 : chaincode initEnclave FPC
+      & Peer2 <-> ecc2 
+      Client1 -> Peer1 : chaincode exportCCKeys FPC
+      & Peer1 <-> ecc1 
+      activate ecc2
```

Comment:
moving `activate ecc2` after `& Peer2 <-> ecc2 ` would look a bit better.

Reply:
yep, was too late, moved it up ..